Update contributing file to meet OSPO requirements (#813)
kyma-project/community: docs/contributing/02-contributing.md

Extend the list whenever you define a new template for other document types. Make sure to update one of the tables in the [**Document types templates**](../guidelines/templates/templates-type.md) document after you add new templates to the `templates/resources` subfolder.

## Developer Certificate of Origin (DCO)

Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this project. SAP uses the [standard DCO text of the Linux Foundation](https://developercertificate.org/).  

> This happens in an automated fashion during the submission process: the CLA assistant tool will add a comment to the pull request. Click it to check the DCO, then accept it on the following screen. CLA assistant will save this decision for upcoming contributions.

Reviewer: ```suggestion
This happens automatically during the submission process: The CLA assistant tool will add a comment to the pull request. The contributer must click it to check the DCO and accept it on the following screen. CLA assistant will save this decision for upcoming contributions.
```

> This DCO replaces the previously used CLA ("Contributor License Agreement") as well as the "Corporate Contributor License Agreement" with new terms which are well-known standards and hence easier to approve by legal departments. Contributors who had already accepted the CLA in the past may be asked once to accept the new DCO.

Reviewer: ```suggestion
This DCO replaces the previously used CLA ("Contributor License Agreement") as well as the "Corporate Contributor License Agreement" with new terms that are well-known standards and hence easier to approve by legal departments. Contributors who had already accepted the CLA in the past may be asked to accept the new DCO.
```

Do we need the "once"? Do we lose any meaning if we drop it?
